libpod/events/logfile.go

```diff
@@ -254,6 +346,10 @@ func renameLog(from, to string) error {
 	}
 	defer fFrom.Close()
 
+	if err := os.Remove(to); err != nil {
+		return fmt.Errorf("recreating file %s: %w", to, err)
+	}
```

Comment:
Is this necessary? The create call should truncate the file anyway.

Reply:
I want to erase any risk of a reader being impacted by the truncation to make sure readers having the file open read to the end.  Am I overthinking?

Reply:
Double checked with Giuseppe who confirmed the need to remove. Truncation would shoot other readers. I added a comment.